wsclient/src/test/resources/RoxieEchoPersonInfo.ecl

```diff
@@ -0,0 +1,3 @@
+import $.esdl_example;
```

Comment:
abandoning this ecl query since it requires esdl_example, opting for the simple ecl query below